src/EnergyPlus/OutputProcessor.cc

```diff
@@ -3997,7 +4002,7 @@ namespace OutputProcessor {
             state.dataOutputProcessor->indexGroupKey = 100;
         } else if (has(meterName, "NaturalGas:Facility")) {
             state.dataOutputProcessor->indexGroupKey = 101;
-        } else if (has(meterName, "DistricHeating:Facility")) {
+        } else if (has(meterName, "DistricHeatingWater:Facility")) {
             state.dataOutputProcessor->indexGroupKey = 102;
```

Comment:
Hmm, this is interesting. I don't know how these `indexGroupKey` values were used in the past, but it appears these are never used. I'm 99% certain all use of this can be removed. Something to note for a followup PR.